### ISSUE TITLE
[BUG][Inspector] Stack won't extracted correctly if browser specific NPM packages are used

### ISSUE THREAD

**Issue by mxschmitt** [microsoft/playwright#5588]
Code:

```js
const { chromium } = require('playwright-chromium');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('http://whatsmyuseragent.org/');
  await page.screenshot({ path: `example.png` });
  await browser.close();
})();
```

Actual:

![image](https://user-images.githubusercontent.com/17984549/108987123-3831f400-7693-11eb-8437-78d0ae8a2fe8.png)

Expected: When using `playwright` instead of `playwright-chromium` it works and shows the correct code in the Inspector window.

`PWDEBUG=1  node test.js`

**Comment by aslushnikov** [comment]
(re-opening this since we'll likely merge this to v1.9.2)

**Comment by aslushnikov** [comment]
This was merged in v1.9.1 and 1.9.1 is live!
